### ISSUE TITLE
Provide relist option before expiration

### ISSUE THREAD

**Issue by kraftbj** [Automattic/WP-Job-Manager#848]
AFAICT, we do not provide an option to relist a expiring listing, only post-expiration. It would be helpful to provide a link within a filterable amount of time before expiration that would allow employer to relist the listing early.

Suggestion: We add the listing period to the end of the listing time. For example, with a 30 day listing, provide a relisting option 5 days before expiration. If renewed at day 25, we add 30 to the initial expiration.

Could allow to be hooked into to notify employers that a listing is expiring.

**Comment by Lens40** [comment]
Yeah would be awesome especially if it works with WC-Paid-Listing...If we have some time we might start some development next year by our self. I keep you posted if we have some ressources available.

**Comment by kochhase** [comment]
Hi Lens40. there is very good plugin, that will help there in many ways. Myles is here as well. See
https://plugins.smyl.es/docs/wp-job-manager-emails/changelog/?v=3a52f3c22ed6

**Comment by Lens40** [comment]
Yeah I have been in contact with Myles about his plugin an some other features (WPML, Multipart-Email and exclusion list). I bought it but did not have the time to set it up yet. But yes it might help us getting that done.

**Comment by spencerfinnell** [comment]
Related #869

**Comment by annezazu** [comment]
599924-zen

**Comment by aheckler** [comment]
https://wordpress.org/support/topic/extend-renew-listings-front-end/

**Comment by Braehler** [comment]
Is this working? I´m looking for this feature for our new jobboard

**Comment by htdat** [comment]
1300338-zen

**Comment by Braehler** [comment]
???

 

Ich seh da nur ne leere Mail^^

 

Von: Dat Hoang [mailto:notifications@github.com] 
Gesendet: Dienstag, 24. Juli 2018 12:30
An: Automattic/WP-Job-Manager
Cc: Braehler; Comment
Betreff: Re: [Automattic/WP-Job-Manager] Provide relist option before expiration (#848)

 

1300338-zen

**Comment by htdat** [comment]
@Braehler - this feature is not implemented yet. 

> 1300338-zen

This is our internal tracking to see if how many users want to have this feature.

**Comment by benjiguth** [comment]
Still waiting on this one. Seriously need it. Any updates?

**Comment by benjiguth** [comment]
Keeping this live, anyone still working on this???

**Comment by iamgabrielma** [comment]
Hi @benjiguth , at this moment this is planned for the 1.3.2 milestone, progress can be seen here: https://github.com/Automattic/WP-Job-Manager/pull/1450

**Comment by benjiguth** [comment]
Amazing thank you!!!

**Comment by benjiguth** [comment]
Just keeping this alive. Any updates??

**Comment by Braehler** [comment]
That would be a feature our customers would love if this is going to be implemented

**Comment by benjiguth** [comment]
Was hoping to see this last year. Still begging for this feature. @htdat @iamgabrielma can you provide any insight here? @Automattic @jom @alexsanford @spencerfinnell @kraftbj @donnapep 
Come on everyone BE THE HEROS I KNOW YOU ARE!!!!!

**Comment by jom** [comment]
This is a high priority, but we don't have an ETA. This change requires coordinated changes with other plugins (such as WooCommerce Paid Courses).

**Comment by benjiguth** [comment]
paid courses or paid listings?

**Comment by jom** [comment]
@benjiguth Sorry--Paid Listings.

**Comment by Braehler** [comment]
Any updates on this?

**Comment by Braehler** [comment]
Any news on that, because this enhancement was not in the latest release. Still realy looking foward to this

**Comment by Braehler** [comment]
One more idea for that feature. When the relist is done, the publish date has to change to the day, when the relist was submitted

**Comment by Braehler** [comment]
@spencerfinnell 
@jom 
Any updates on this one here? Waiting for this for such a long time. A realy needed feature

**Comment by benjiguth** [comment]
> @spencerfinnell
> @jom
> Any updates on this one here? Waiting for this for such a long time. A realy needed feature

I am also waiting patiently on this one.

**Comment by alexsanford** [comment]
Unfortunately, we aren't able to focus on this at the moment 🙁 

There is an (old) open PR that was started here - https://github.com/Automattic/WP-Job-Manager/pull/1450

If anyone wants to pick this up, or create a new PR, we are happy to accept contributions.

**Comment by Braehler** [comment]
Hey,
sad to hear that you do not set a focus on that. This is a realy needed feature, that would make many customers happy.
Any kind of roadmap for the next releases including some new features?

**Comment by cena** [comment]
3297703-zen

**Comment by Braehler** [comment]
Any news on this one? Long avaited feature

**Comment by gaurav1984** [comment]
5317098-zen

**Comment by lxmultimedia** [comment]
Are there any news on this feature...? Long awaited indeed... thank you

**Comment by michaelnutt2** [comment]
5877209-zen
